docs/get-started.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+Create your own helm broker bundles list by fork bundles repository.
+
+## Prerequisites
+
+Before you start install:
+
+* [Kyma](https://kyma-project.io/docs/latest/root/kyma#getting-started-getting-started)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [svcat CLI](https://svc-cat.io/docs/cli/)
+
+## Steps
+
+### Create fork and generate personal access token
+
+1. Create fork from the [bundle repository](https://github.com/kyma-project/bundles)
+2. On your forked repository create `gh-pages` branch if not exist
+3. Go to **Settings** tab and in section **GitHub Pages** setup **Source** to `gh-pages branch`
+4. Go to **Settings** tab on your [Github profile account](https://github.com/settings/profile), in the left sidebar, click **Developer settings**.
+5. In the left sidebar, click **Personal access tokens**.
+6. Click **Generate new token**.
+7. Give your token a descriptive name.
+8. Select scope `repo -> public_repo` (Access public repository)
+9. Click **Generate token**.
+
+> **NOTE:** Save token before you close browser tab
+
+### Modify bundle yaml file
+
+1. On your forked repository, modify `index.yaml` file, for example you can edit description in `bundles/stable/index.yaml` file
+2. Instructions how to create custom bundle you find in [Kyma documentation](https://kyma-project.io/docs/latest/components/service-brokers#configuration-how-to-create-a-ybundle)
+
+### Integrate Travis-CI with Github repository
+
+1. Go to [travis-ci](https://travis-ci.com/) and login using your Github account
+2. Add access to your forked `bundle` repository.
+3. In the left sidebar, click **My Repositories** tab and choose your Github repository
+4. In the right tab click **More options** then **Settings**
+5. Go to **Environment Variables** section and add key-value pair where key is `GITHUB_TOKEN` and value is equal to your Github personal access token
+6. In the right tab click **More options** then **Trigger build** and wait for Travis to finish his work
+
+> **NOTE:** To make sure your Github pages works properly go to your Github repository, click **Settings** tab, then go to **GitHub Pages** section and click on link next to **Your site is published at** phrase
+
+### Setup Kyma Helm Config Map
+
+1. Go to your terminal and edit Helm ConfigMap base url with your Github pages ulr
+```
+> kubectl set env -n kyma-system deployment/core-helm-broker -e APP_REPOSITORY_BASE_URL=https://user-name.github.io/bundles/stable/
+```
+> **NOTE:** To be sure you have this config map you can display all config maps `kubectl get cm -n kyma-system`
+
+2. Wait until kubernetes restart `core-helm-broker` Pod
+
+> **NOTE:** Check if Pod is running by `kubectl get pod -n kyma-system -l app=core-helm-broker` command
+
+3. Synchronize Service Catalog
+```
+svcat sync broker core-helm-broker
+```
+
+### Check your Kyma Servise Catalog
+
+1. Go to Kyma console in your browser
+2. In the left sidebar, click **Catalog**
```

Comment:
and then?

IMO we should add sth like. 
The new bundles should appear. If not then go to the Troubleshooting section.

and in troubleshooting add e.g. how to get the logs from the helm-broker Pod because there should be information about incorrectly indexed bundles. Additionally how to execute the checker tool against custom bundles.